velox/docs/functions/spark/coverage.rst

```diff
@@ -0,0 +1,129 @@
+=================
+Function Coverage
+=================
+
+Here is a list of all scalar and aggregate spark functions with functions that are available in Velox highlighted.
```

Comment:
How did you generate that list? Would you add a readme with instructions ?

Reply:
fixed.